src/librustdoc/html/render.rs

```diff
@@ -606,12 +606,20 @@ pub fn run(mut krate: clean::Crate,
 }
 
 // A short, single-line view of `s`.
-fn concise_str(s: &str) -> String {
+fn concise_str(mut s: &str) -> String {
     if s.contains('\n') {
-        return format!("{}...", s.lines().next().expect("Impossible! We just found a newline"));
+        s = s.lines().next().expect("Impossible! We just found a newline");
     }
     if s.len() > 70 {
-        return format!("{} ... {}", &s[..50], &s[s.len()-20..]);
+        let mut lo = 70;
```

Comment:
This should be 45 to make sure the resulting string is at most 70 characters.

Reply:
Addressed this, thanks! I set to 50 (as in the old code), there is no strict need to be under 70 chars, it is just roughly a number that should fit on one line (comfortably under 80 chars).

Reply:
Well that's okay but if the line is exactly 71 characters for example then using 50 will skip one character and replace it with `" ... "` making the string display actually longer. It's only a minor thing though.